### PR TITLE
#77 Publish locally and add only substrate and transitive dependencie…

### PR DESCRIPTION
**Pull request by jperedadnr**
…s to classpath

For now it has a hard dependency on the snapshot version

### CODE REVIEW

File: test-project/helloFX/build.gradle [hunk 1]
```diff
@@ -3,17 +3,18 @@ plugins {
...
 
 repositories {
+    mavenLocal()
     mavenCentral()
 }
 
 def target = System.getProperty("substrate.target")
 
+configurations {
+    provided
```
Comment (tiainen) [inline test-project/helloFX/build.gradle]: `substrate` might be a better name?

File: test-project/helloFX/build.gradle [hunk 2]
```diff
@@ -3,17 +3,18 @@ plugins {
...
+}
+
 dependencies {
-    if ("ios" == target) {
-        implementation 'com.googlecode.plist:dd-plist:1.22'
-        implementation 'com.github.jnr:jnr-ffi:2.1.10'
-        implementation 'org.bouncycastle:bcpkix-jdk15on:1.49'
-    }
+    provided "com.gluonhq:substrate:0.0.2-SNAPSHOT"
```
Comment (tiainen) [inline test-project/helloFX/build.gradle]: Are the other dependencies no longer needed on the classpath?

File: test-project/helloFX/build.gradle (hunk 1 above)
Comment (jperedadnr) [reply test-project/helloFX/build.gradle]: ok

File: test-project/helloFX/build.gradle (hunk 2 above)
Comment (jperedadnr) [reply test-project/helloFX/build.gradle]: These are transitive dependencies, and therefore, they are included in the classpath

Comment (tiainen) [reply test-project/helloFX/build.gradle]: True.

**Pull request merged**